packages/react-components/src/components/MessageThread.test.tsx

```diff
@@ -225,3 +225,290 @@ describe('Message should display inline image correctly', () => {
     });
   });
 });
+
+/* @conditional-compile-remove(mention) */
+describe('Message should display Mention correctly', () => {
+  beforeAll(() => {
+    registerIcons({
+      icons: {
+        chatmessageoptions: <></>,
+        messageedit: <></>,
+        messageremove: <></>,
+        messageresend: <></>,
+        editboxcancel: <></>,
+        editboxsubmit: <></>
+      }
+    });
+  });
+
+  test('Message should include Mention', async () => {
+    const user1Id = 'user1';
+    const user2Id = 'user2';
+    const user2Name = 'Robert Tolbert';
+
+    const messages: ChatMessage[] = [
+      {
+        messageType: 'chat',
+        senderId: user1Id,
+        senderDisplayName: 'Kat Larsson',
+        messageId: Math.random().toString(),
+        content: `Hey <msft-mention id="${user2Id}" displayText="${user2Name}"></msft-mention>, can you help me with my internet connection?`,

```

Comment:
I think we need to have the display name inserted between the open and closing tag:
`Hey <msft-mention id="${user2Id}" displayText="${user2Name}">{user2Name}</msft-mention>, can you help me with my internet connection?`

Reply:
Technically we don't need it between as part of the message content, as it will parse the `id` and `dispalyText` from the `msft-mention` attribute, then in the `defaultOnMentionRender` it will inject the proper tag with the `displayText` also in the content to be:
```
<msft-mention id="${user2Id}" displayText="${user2Name}">{user2Name}</msft-mention>
```
I purposely omit it so I can assert/expect the displayText is injected properly as part of `msft-mention` textContent

Reply:
This is actually a little bit of a strange behaviour. The text between the tag should be rendered and in the SendBox, the enclosed text is used as the displayed text, ignoring the display text parameter.

Reply:
So as we're dropping the displayText parameter and using the text in the tags, we should have our tests reflect that.
It's also important to remember that browsers will render any custom tag in this way - using the content between the tags rather than the parameter - so we should be consistent

Reply:
so now that we walked back that change, we _do_ need the tag content to contain the displayText - and we should ensure that this is the case when rendered